projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -1159,15 +1165,22 @@ export class IgxTimePickerComponent implements
         if (this.collapsed) {
             this.collapsed = false;
 
-            let settings;
+            let settings, posStrategy;
```

Comment:
`posStrategy` is not assigned more than once so no need to declare it here. You could declare it in line 1175.